include/boost/geometry/algorithms/detail/sections/sectionalize.hpp

```diff
@@ -778,8 +778,49 @@ struct sectionalize_multi
     }
 };
 
-template <typename Sections>
-inline void enlarge_sections(Sections& sections)
+// TODO: If it depends on CS it should probably be made into strategy.
```

Comment:
I don't think it should be a strategy.

As far as I can see, just increasing the sections by a larger amount than just an epsilon is OK for all coordinate systems. Because otherwise some intersections can be missed in rare cases.
Enlarging the sections does no harm, other than that it might compare a few segment pairs more.

Can't you just expand by a larger value, regardless of rescaling or coordinate system? Did you try that? To me that sounds better than introducing this strategy.

Reply:
Yes right now, enlarging them by greater value should work. Providing various strategies allows to tweak enlargement values for specific formulas. It is not needed to pay the performance price in cartesian because andoyer formula has low accuracy in geographic.

However, I'm considering a possibility of passing epsilon used e.g. in point comparisons or distance comparisons in a strategy. It's because various formulas have different accuracy. If this was possible then a user would be able to define the epsilon too to something even greater and then in order to be consistent it should be possible to change the enlargement value as well.

Reply:
OK, that sounds interesting and more generic.